src/Analysis/Ast/Impl/Analyzer/Evaluation/ExpressionEval.Collections.cs

```diff
@@ -23,6 +23,8 @@
 
 namespace Microsoft.Python.Analysis.Analyzer.Evaluation {
     internal sealed partial class ExpressionEval {
+        private const int MaxCollectionSize = 1000;
```

Comment:
This may be made settable for tests for you can set it to 5 and make a test for it. Up yo you.

Reply:
Yeah, this is somewhat deep in the analysis so there's no good direct way to test it (given you need to provide it a module and such). I'll just leave it as-is for now.